reporting/config/services/superset/app-customizations/0.29.0rc7/assets/src/dashboard/components/Header.jsx

```diff
@@ -0,0 +1,330 @@
+/* eslint-env browser */
```

Comment:
Can you mark with comments which lines were changed from the original version?

Reply:
Done.